### DISCUSSION TITLE
How can I send a inline link button?

### DISCUSSION THREAD

**Discussion by codaliza** [kianahmadian/python-bale-bot#10] [Q&A] [answered]
I have a photo conversion bot, and I want to send a link to the user after the photo conversion is done.
Can I send a link with a button?

**Comment by kian-ahmadian** [comment, marked as answer]
Hi @erfan-s, Yes it's possible.

> for example:

```py
...
async def on_message(message):
     await message.reply(..., components = bale.Components(inline_keyboards = [[bale.InlineKeyboard(text = "Link", url = "https://github.com/python-bale-bot")]] ))
```

You can also read this in the [project document](https://python-bale-bot.readthedocs.io/en/master/api.html#bale.InlineKeyboard).

Enjoy! 🤗
